### ISSUE TITLE
ADR 2.0.1 <pre> URI's have wrong highlighting creating A11Y contrast issues

### ISSUE THREAD

**Issue by bwbroersma** [Geonovum/KP-APIs#642]
Currently `<pre>URI</pre>` is used, see [DesignRules.md#L51](https://github.com/Logius-standaarden/API-Design-Rules/blob/e207f4c983b2e3eb068a416976c999ab3916f3c4/DesignRules.md?plain=1#L51):
```html
<pre>https://api.example.org/v1/gebouwen<br/>https://api.example.org/v1/vergunningen</pre>
```

In the published [ADR 2.0.0](https://gitdocumentatie.logius.nl/publicatie/api/adr/2.0.0/#/core/naming-collections) this results in no contrast issues, but highlightjs will wrongly auto detected this as JavaScript, so `https:` as label and the following `//` as comment:
```html
<code class="hljs javascript">https:<span class="hljs-comment">//api.example.org/v1/gebouwen</span>
https:<span class="hljs-comment">//api.example.org/v1/vergunningen</span></code>
```
There is no contrast issue in ADR 2.0.0, but in [ADR 2.0.1](https://gitdocumentatie.logius.nl/publicatie/api/adr/2.0.1/#/core/naming-collections) is probably generated with a newer highlightjs version, since the generated HTML now also has the `hljs-attr` styling class:
```html
<code class="hljs javascript"><span class="hljs-attr">https</span>:<span class="hljs-comment">//api.example.org/v1/gebouwen</span>
<span class="hljs-attr">https</span>:<span class="hljs-comment">//api.example.org/v1/vergunningen</span></code>
```
Together with a color change, the contrast of the 'comment part' of the URI against the background is 2.32, which is too low for WCAG AA rating.

Maybe the [`nohighlight`](https://respec.org/docs/#nohighlight) class should be used? E.g.
```html
<pre class="nohighlight">https://uri</pre>
```

BTW If your unfamiliar, see the Firefox Developer Tools, Accessibility tab and check for contrast:
![Screenshot of Firefox Developer Tools Accessibility tab](https://github.com/user-attachments/assets/89d55408-6367-4858-91ba-5ec6a2672658)
Also see [MDN about WCAG ratings](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast) for more information.
